integration/query_bitwise_test.go

```diff
@@ -98,8 +99,9 @@ func TestQueryBitwiseAllClear(t *testing.T) {
 			err: &mongo.CommandError{
 				Code:    2,
 				Name:    "BadValue",
-				Message: "value takes an Array, a number, or a BinData but received: $bitsAllClear: \"123\"",
+				Message: `v takes an Array, a number, or a BinData but received: $bitsAllClear: "123"`,
 			},
+			altMessage: `value takes an Array, a number, or a BinData but received: $bitsAllClear: "123"`,
```

Comment:
That's a small bug in our implementation ("value" there should be a field name, but it is a fixed string in our code), but we are not fixing it now. See #508.